packages/web-components/src/components/va-additional-info/va-additional-info.tsx

```diff
@@ -23,17 +23,17 @@ export class VaAdditionalInfo {
   /**
    * The text to trigger the expansion
    */
-  @Prop() trigger: string;
+  @Prop() trigger!: string;
```

Comment:
Actually, since we're doing things like this, is there something we can do in the helper function to make that red asterisk appear for required props?

![trigger prop in additional-info component not showing as required in storybook](https://user-images.githubusercontent.com/2008881/176731365-bef81449-c810-4051-915c-97d4b649de92.png)